nekoyume/Assets/Editor/OpenAPIGenerator.cs

```diff
@@ -891,7 +884,7 @@ private static void AppendWebRequestsFromPaths(StringBuilder sb, JsonNode pathsN
 
                         sb.AppendLine($"    public async Task {methodName}({parameterDefinitions}Action<{returnType}> onSuccess, Action<string> onError)");
                         sb.AppendLine("    {");
-                        sb.AppendLine($"        string url = Url + \"{path}\";");
+                        sb.AppendLine($"        string url = Url + $\"{path}\";");
```

Comment:
```suggestion
                        sb.AppendLine($"        string url = $\"{Url}{path}\";");
```
문자열 보간 쓰는 겸 Url 스트링도 안으로 넣는게 좋을 것 같습니다

Reply:
스트링으로 적는 Url은 런타임시 세팅되는 값을 의미하는거라서 코드 생성시점에 정할수가없어서 이렇게 처리했습니다~!

Reply:
에디터 타임에서 텍스트가 자동생성 되는 타이밍에 달러 스트링 안으로 Url이 들어가는 형태로 수정되게 할 수는 없을까요?>

Reply:
url은 다양한 운영요소로 바뀌게되어서 생성타임에 url을 고정하는것은 안좋은선택인것같아요
단적인 예로 프로토콜은 동일하나, 헤임달url과 메인넷 url이 다를 경우 런타임 수정이 불가하게됩니다~!

Reply:
아뇨 런타임에 바뀔 수 있게 구성하는건 동일한데, Url자체가 문자열 보간으로 변경될 수 있도록 하면 좋을 것 같아서 했던 말입니다

Reply:
확인감사합니다~!
https://github.com/planetarium/NineChronicles/pull/6091/commits/e88947db170584deb255363c624541f7ea722734
에서 수정진행하였습니다~